### PR TITLE
Update de-de.php

### PR DESCRIPTION
**Pull request by WebCrew**
### Pull Request Summary

I did a few updates and wrote some infos about "Slug oder Rohling" and about "Sie oder Du". By the way, I`m from Germany, so I have a little knowledge about German (lol)

Kind regards, Andy

### Have you...

- [x] Checked that your code adheres to our code guidelines?

---

### Contributor License Agreement

We want Leafpub to remain free and open for everyone to use forever, so please don’t submit any code or content that is licensed or copyrighted without first getting written consent.

Before we accept your pull request, we ask that you agree to the following terms to ensure there are no conflicts with Leafpub’s license now or in the future.

1. I certify that the code I am contributing to this project is completely original or free of any known license or copyright.

2. I agree to allow the developer of Leafpub license or relicense my work at their discretion. By submitting this pull request, I agree to forfeit all copyrights, intellectual property rights, and royalties pertaining to my contribution.

3. By entering my name in the space provided below, I agree to all of the terms mentioned above.

Signed: [Andreas Holzer]

Date: [2017-01-10]

### CODE REVIEW

File: app/source/languages/de-de.php [hunk 1]
```diff
@@ -5,7 +5,7 @@
     '1_post' => '1 Post',
     'a_password_reset_has_been_requested_for_this_account' => 'Eine Passwortzurücksetzung wurde für dieses Konto angefordert:',
     'a_post_already_exists_at_the_url_you_entered' => 'Es existiert bereits ein Post unter der eingegebenen URL',
-    'a_tag_already_exists_at_the_slug_you_entered' => 'Es existiert bereits ein Schlagwort unter dem eingegebenen Rohling',	// slug
+    'a_tag_already_exists_at_the_slug_you_entered' => 'Es existiert bereits ein Schlagwort unter dem eingegebenen Slug',	// slug ist besser und verbreiteter als "Rohling"
```
Comment (karsasmus) [inline app/source/languages/de-de.php]: I would suggest to change the german sentence to "Dieses Schlagwort existiert bereits".

File: app/source/languages/de-de.php [hunk 2]
```diff
@@ -183,7 +183,7 @@
     'select_blog_plattform' => 'Wähle eine Blog Plattform',
     'set_post_image' => 'Postbild setzen',
     'settings' => 'Einstellungen',
-    'share_a_sentence_or_two_about_yourself' => 'Schreiben Sie ein bis zwei Sätze über sich',	// Sie oder du?
+    'share_a_sentence_or_two_about_yourself' => 'Schreiben Sie ein bis zwei Sätze über sich',	// "Sie" ist verbreiteter und üblich in Deutschland. Du sagt man zu Freunden und Bekannten.
     'slug' => 'Rohling',
```
Comment (karsasmus) [inline app/source/languages/de-de.php]: How about "Kurzwort"? Setzmaschinenzeile, Schnecke or Rohling isn't a suitable translation, you're right.

File: app/source/languages/de-de.php [hunk 3]
```diff
@@ -205,7 +205,7 @@
     'the_password_you_entered_is_not_valid' => 'Das eingegebene Passwort ist nicht gültig',
     'the_password_you_entered_is_too_short' => 'Das eingegebene Passwort ist zu kurz',
     'the_passwords_you_entered_do_not_match' => 'Die eingegebenen Passswörter stimmen nicht überein',
-    'the_slug_you_entered_cannot_be_used' => 'Der eingegebene Rohling kann nicht verwendet werden',	// slug
+    'the_slug_you_entered_cannot_be_used' => 'Der eingegebene Slug kann nicht verwendet werden',	// "slug" ist besser als "Rohling"
```
Comment (karsasmus) [inline app/source/languages/de-de.php]: If we translate "slug" as "Kurzwort" then we should edit "Slug" to "Kurzwort". What are your thoughts about the change?

File: app/source/languages/de-de.php (hunk 2 above)
Comment (WebCrew) [inline app/source/languages/de-de.php]: Yeah "Kurzwort" is also nice, even better as "slug"

File: app/source/languages/de-de.php (hunk 3 above)
Comment (WebCrew) [inline app/source/languages/de-de.php]: good idea, makes sense

Comment (karsasmus) [inline app/source/languages/de-de.php]: That's the correct case, of course :smile:

File: app/source/languages/de-de.php (hunk 1 above)
Comment (WebCrew) [inline app/source/languages/de-de.php]: Sounds good ;)

P.S.: Have you done the whole German translation? If yes, thumbs up - its done very good.

Comment (karsasmus) [inline app/source/languages/de-de.php]: I haven't done the inital translation. But I changed and added a lot. ( By the way I live in the wonderfull Saarland ;) )

Comment (WebCrew) [inline app/source/languages/de-de.php]: ahhhhhhh you live in Saarland - Völklingen - nice

**Pull request merged**